test/generation_test_setup/dummy_page_loader_element.dart

```diff
@@ -232,3 +233,9 @@ class DummyPageLoaderElement implements PageLoaderElement {
   @override
   Future<Null> blur() => throw 'not implemented';
 }
+
+class DummyPageLoaderAttributes extends DelegatingMap<String, String>
+    implements PageLoaderAttributes {
+  @override
+  final delegate = <String, String>{};
```

Comment:
Any concern about this being writable?

Reply:
It needs to be writable. It's just a mock being used in tests so it should be fine.